components/Tooltip/Tooltip.stories.tsx

```diff
@@ -35,3 +67,77 @@ MultiLine.args = {
   content:
     'This is some tooltip text. This box shows the max amount of text to display. If more room is needed, use a modal instead.',
 };
+
+export const NodeContent = Template.bind({});
+
+const WarningOption = <Flex align="center" gap={2}><ExclamationTriangleIcon /><Text css={{ color: 'currentColor' }}>Warning message</Text></Flex>
+
+NodeContent.args = {
+  content: WarningOption
+};
+
+NodeContent.argTypes = {
+  content: {
+    options: ['Warning', 'Disabled', 'Heading'],
+    mapping: {
+      Warning: WarningOption,
+      Disabled: <Flex align="center" gap={2}><CrossCircledIcon /><Text css={{ color: 'currentColor' }}>Disabled message</Text></Flex>,
+      Heading: <Flex align="center" gap={2}><Text css={{ fontWeight: 700, color: 'CurrentColor' }} >Heading</Text><Text css={{ color: 'currentColor' }}>Content</Text></Flex>,
```

Comment:
I think it could be great to also move those to variables, it will be more readable IMO.